tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "@octokit/tsconfig",
   "compilerOptions": {
-    "module": "CommonJS",
+    "module": "Node16",
```

Comment:
I think with the updates of Octokit we now only support Node 18+?

Reply:
Why dont we use the default value of @octokit/tsconfig?

Reply:
We do.
The name is kinda misleading, it for ESM + CJS support in a Node 16+ environment.

https://www.typescriptlang.org/docs/handbook/modules/reference.html#node16-nodenext

Reply:
Yes, it's needed for ESM support for `probot run`, otherwise TypeScript tries to transpile `await import()` calls.

There was a hack I used previously with eval, but that broke Jest, and this makes the output cleaner.